src/browser/index.jsx

```diff
@@ -466,29 +466,67 @@ var MattermostView = React.createClass({
 });
 
 // ErrorCode: https://code.google.com/p/chromium/codesearch#chromium/src/net/base/net_error_list.h
-// FIXME: need better wording in English
+var errorPage = {
+  tableStyle: {
+    display: 'table',
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+    top: '-70px',
+    left: '0'
+  },
+
+  cellStyle: {
+    display: 'table-cell',
+    verticalAlign: 'middle'
+  },
+
+  bullets: {
+    paddingLeft: '15px',
+    lineHeight: '1.7'
+  },
+
+  techInfo: {
+    fontSize: '12px',
+    color: '#aaa'
+  },
+};
+
 var ErrorView = React.createClass({
   render: function() {
     return (
       <Grid id={ this.props.id } style={ this.props.style }>
-        <h1>Failed to load the URL</h1>
-        <p>
-          { 'URL: ' }
-          { this.props.errorInfo.validatedURL }
-        </p>
-        <p>
-          { 'Error code: ' }
-          { this.props.errorInfo.errorCode }
-        </p>
-        <p>
-          { this.props.errorInfo.errorDescription }
-        </p>
-        <p>Please check below. Then, reload this window. (Ctrl+R or Command+R)</p>
-        <ListGroup>
-          <ListGroupItem>Is your computer online?</ListGroupItem>
-          <ListGroupItem>Is the server alive?</ListGroupItem>
-          <ListGroupItem>Is the URL correct?</ListGroupItem>
-        </ListGroup>
+        <div style={ errorPage.tableStyle }>
+          <div style={ errorPage.cellStyle }>
+            <Row>
+              <Col xs={ 0 } sm={ 1 } md={ 1 } lg={ 2 } />
+              <Col xs={ 12 } sm={ 10 } md={ 10 } lg={ 8 }>
+              <h2>Cannot connect to Mattermost</h2>
+              <hr />
+              <p>We're having trouble connecting to Mattermost. If refreshing this page (Ctrl+R or Command+R) does not work please verify that:</p>
+              <br />
+              <p>
```

Comment:
The console shows `index.js:1812 Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.` In my setting, which has three teams and the third is incorrect URL, I could not click the second tab while showing the third tab. Is this related?